### PR TITLE
Adapt web to pwa 

### LINKED ISSUE #25: Adapt app to other platforms
Description:
We want to adapt the current web based project to native apps (Desktop, iOS, PWA) to increase user retention. 
Requirements:
Research the feasibility of adapting to each platform, pay attention to the difficulty of implementation (can we finish within one sprint?)
Deliverable:
Report on feasibility, tools we need to use, high-level steps to adapt for each platform.

### PR DESCRIPTION
**Pull request by kgeewhan**
adapts web app to a pwa

### CODE REVIEW

File: cypress/integration/main/keystroke.spec.js [hunk 1]
```diff
@@ -9,6 +9,15 @@ const SETTINGS_SLIDE_CLOSE = 'slide-close-settings';
 
 describe('Timer Keys Test', ()=>{
     beforeEach(() => {
+        // unregisters service worker before e2e test
```
Comment (friidryce) [inline cypress/integration/main/keystroke.spec.js:13]: Could possibly encapsulate this into a function to eliminate code repetition

**Review by friidryce — APPROVED** [review]
Good work on adapting to PWA and making the adjustments to Cypress testing to accommodate

File: cypress/integration/main/keystroke.spec.js (hunk 1 above)
Comment (kgeewhan) [reply cypress/integration/main/keystroke.spec.js:13]: made this into a function in sw.js

**Review by cbj99 — APPROVED** [review]
good changes, great work

**Pull request merged**